### ISSUE TITLE
App spacers -- trying to add but nothing happens?

### ISSUE THREAD

**Issue by Bmuse21** [w0lfschild/cDock#70]
OS X 10.11.5
cDock 2: 0.10.1

Hi,

I'm trying to add app spacers in cDock 2. When I increase the number and hit apply, nothing happens.

Where are they supposed to appear? And once they appear, do you drag them where you want them?

Other customizations work, but I can't get any app spacers to appear.

Thank you for any assistance!

**Comment by w0lfschild** [comment]
Could be something to do with your Dock plist being broken.

Try updating to the latest version of cDock which is 0.11.2 https://github.com/w0lfschild/app_updates/blob/master/cDock/cDock_0.11.2.zip

**Comment by Bmuse21** [comment]
Hello, and thank you for the reply!

I updated to 0.11.2.  Removed the Dock plist. Removed and reinstalled cDock 2. SIP is disabled.

Theming works. The theme changes when I select one in the dropdown on the Theming tab.

But nothing on the Dock tab works. If I check/change anything and enter a value, then hit Apply, it doesn’t happen. Select the dock position and click Apply, it doesn’t change. Add spacers and click Apply, nothing happens.

If I do something on the Dock tab, hit Apply, then switch to the Theming tab and restart the dock, all the Dock tab values reset to default.

The cdock.bundle is in the SIMBL plugins folder, and that folder has root level read/write permissions.

I have no idea what else to check?

Thank you for your help, I would LOVE to get cDock 2 working properly!
